### ISSUE TITLE
Please consider adding Compton

### ISSUE THREAD

**Issue by MattiaVerticchio** [clearlinux/distribution#984]
As I wrote [on the forum](https://community.clearlinux.org/t/how-to-enable-tearfree/1011?u=mattiaverticchio), a common problem of i3 is the presence of tearing, especially when scrolling or watching videos.

As far as I know, Clear uses the `modesetting` driver – which doesn't have a built-in TearFree option. A solution for this issue could be including the [Compton compositor](https://github.com/yshui/compton) in `desktop-i3`, or providing it as a standalone bundle.

**Comment by bryteise** [comment]
Hrm so it appears that Compton fork is still in an experimental/unstable mode. I'm not really sure about grabbing the vNext pre-release from May though it does have a few bug fixes that seem like they could be critical. Any opinions here?

**Comment by MattiaVerticchio** [comment]
I have no technical info about the pre-release version stability. For this reason I would prefer the [6.2 one](https://github.com/yshui/compton/releases/tag/v6.2), which is the same included in the [Arch Community repository](https://www.archlinux.org/packages/community/x86_64/compton/).

**Comment by nsgnn** [comment]
I would agree that v6.2 would be a good addition.

**Comment by MattiaVerticchio** [comment]
Compton has been recently updated to [v7.2](https://github.com/yshui/compton/releases).

**Comment by 0n0w1c** [comment]
Please consider making a separate bundle for compton. I would like to use it with desktop-lxqt.

**Comment by lebensterben** [comment]
>Please consider making a separate bundle for compton. I would like to use it with desktop-lxqt.

@0n0w1c I agree. Because one can also use compton for dwm, which by its nature is compiled by the user himself and not likely to be packaged.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by nsgnn** [comment]
Any updates on this? It seems to not be live yet.

**Comment by bryteise** [comment]
Looks like upstream became unmaintained and there is now a picom fork.
